src/EnergyPlus/PollutionModule.cc

```diff
@@ -6185,7 +6185,7 @@ void ReadEnergyMeters(EnergyPlusData &state)
     // This routine reads the meters for the energy used
 
     // Using/Aliasing
-    auto &FracTimeStepZone = state.dataHVACGlobal->FracTimeStepZone;
+    Real64 FracTimeStepZone = state.dataHVACGlobal->FracTimeStepZone;
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     auto &Pollution = state.dataPollutionModule->Pollution;
```

Comment:
auto const &Pollution?

Reply:
CppCheck agrees with you!

![image](https://user-images.githubusercontent.com/5778156/224459528-aa38c516-4e5b-4144-bca1-88b24d636476.png)